metadataStore/commands.py

```diff
@@ -1,93 +1,89 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
-
 import six
-
-
-__author__ = 'arkilic'
-
-from metadataStore.database.header import Header
-from metadataStore.database.beamline_config import BeamlineConfig
-from metadataStore.database.event_descriptor import EventDescriptor
-from metadataStore.database.event import Event
+from metadataStore.odm_templates import (BeginRunEvent, BeamlineConfig,
+                                         EndRunEvent, EventDescriptor, Event)
 import datetime
 from metadataStore.conf import host, port, database
 from mongoengine import connect
-import time
-# TODO: Add logger
 
-# TODO: Update docs after refactoring
 
-
-def save_header(event_descriptors, beamline_config, unique_id,
-                scan_id, create_time, **kwargs):
-    """Create a header in metadataStore database backend
+def insert_begin_run(time, beamline_id, beamline_config=None, owner=None,
+                     scan_id=None, custom=None):
+    """ Provide a head for a sequence of events. Entry point for an
+    experiment's run.
 
     Parameters
     ----------
-    scan_id : int
-        scan identifier visible to the user and data analysis.
-        Auto incremented by opyhd
-
-    event_descriptors : list
-        List of EventDescriptor objects the header is going to contain
-
-    beamline_config : metadataStore.database.beamline_config.BeamlineConfig
-        BeamlineConfig object used to capture events given headers
-        and event_descriptors
-
-    unique_id : str
-        String field containing unique hash generated by ophyd
-
-    create_time : float
-        Creation time of a Header
-
-
-    Other Parameters
-    ----------------
-
-    owner : str, optional
-        Specifies the unix user credentials of the user creating the entry
-
-    beamline_id : str, optional
+    time : float
+        The date/time as found at the client side when an event is
+        created.
+    beamline_id: str
         Beamline String identifier. Not unique, just an indicator of
         beamline code for multiple beamline systems
-
-    custom : dict, optional
+    owner: str, optional
+        Specifies the unix user credentials of the user creating the entry
+    scan_id : int, optional
```

Comment:
I need clarification on scan_id. It is optional anyway. It is a simple field change from my perspective.